modules/images-s2i-build-process-overview.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-s2i-build-process-overview_{context}"]
+= Source-to-image build process overview
+
+Source-to-image (S2I) produces ready-to-run images by injecting source code into a container and letting the container prepare that source code for execution. It performs the following steps:
+
```

Comment:
Are these steps also out of date? Should they be the same as in images-create-s2i-build.adoc:

S2I generates a Dockerfile with the builder image as the first `FROM` instruction. The Dockerfile generated by S2I is then passed to Buildah.

Reply:
Correct - s2i generates a Dockerfile with the following instructions:

- `FROM <builder image>`
- Copies the source code to a defined location in the builder image
- Runs the assemble script in the builder image
- Sets the run script in the builder image as the default command.

Buildah then creates the container image with this Dockerfile.